CMakeLists.txt

```diff
@@ -1,7 +1,6 @@
 cmake_minimum_required(VERSION 3.2)
 
-message(STATUS "Configuring ${CMAKE_BUILD_TYPE} build of ChiTech.")
```

Comment:
Was there a reason to remove this line? I added it only recently, just to make it more transparent to the user what is being built.